pipelines/SNP-genotyping-vector/farm5/SNPGenotyping.wdl

```diff
@@ -52,4 +52,7 @@ workflow SNPGenotyping {
     File output_vcf_index = UnifiedGenotyper.output_vcf_index
     File zarr_output = VcfToZarr.zarr_output
   }
+  meta {
+    allowNestedInputs: true
```

Comment:
Is this just for metadata labelling?

Reply:
Or is it consumed by another process?

Reply:
This allows you to specify a parameter to a task, without threading it all the way through to be an input to the main WDL.  So you can say something like: 'ShortReadAlignment.ValidateSamFile.num_cpu = 2' (without having to make num_cpu be an input to ValidateSamFile).